### PR TITLE
Addition of preferential diffusion model for flamelet solver

### PR DESCRIPTION
**Pull request by EvertBunschoten**
Addition of tutorial files demonstrating the SU2 flamelet solver with preferential diffusion according to the added features in PR #2156

### CODE REVIEW

**Review by pcarruscag — COMMENTED** [review]
Is this tested in the tutorials.py regression?

**Comment by EvertBunschoten** [comment]
> Is this tested in the tutorials.py regression?

Ah shoot, forgot about that one. Now it's in the list.

**Comment by bigfooted** [comment]
Looks good...

File: incompressible_flow/Inc_Combustion/1__premixed_hydrogen/H2_burner.cfg [hunk 1]
```diff
@@ -0,0 +1,144 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% SU2 configuration file                                                       %
+% Case description: Laminar premixed flame stabilized on isothermal burner     %
+% Author: Nijso Beishuizen                                                     %
+% Institution: Bosch Thermotechnology                                          %
```
Comment (bigfooted) [inline incompressible_flow/Inc_Combustion/1__premixed_hydrogen/H2_burner.cfg]:
```suggestion
% Case description: Laminar premixed flame stabilized on isothermal burner     %
% Author: Evert Bunschoten                                                     %
% Institution: Delft University of Technology                                  %
```

**Review by bigfooted — APPROVED** [review]
LGTM

**Pull request merged**